ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```diff
@@ -0,0 +1,22 @@
+binary_build_commands: make manager
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: "true" # TODO `make unit-test` once the code is there
```

Comment:
I will suggest:
1. Add the real command as from now (and make the test optional)
2. instead of running `make unit-test`, run `script/release/unit_test.sh` same as we are doing it k8s prow https://github.com/ybettan/test-infra/blob/d9ddfca68c0b2d219ce33bf9f2fbb2613545690c/config/jobs/kubernetes-sigs/special-resource-operator/special-resource-operator-presubmits.yaml#L62

This approach allows us to:
1. create PRs that have some code and also fix the CI accordingly in the same PR (instead of fist pushing some code with broken CI and then go fix the CI in a different PR)
2. align with the CI we have for NFD in kube
3. less changes to the `release` repo - no need to be familiar with the CI infra in order to change some tests.

Reply:
PTAL.

Reply:
I still see some makefile commands, is this intended?